README.md

```diff
@@ -73,3 +75,12 @@ be a fork.
 
 * It's up to you whether or not to rename your fork/own version. Do whatever
   you think sounds good.
+  
+## Best Practices
+
+Jina apps created with the methods above:
+
+- Store their Pods in the `pods` directory
```

Comment:
I am not sure I like this idea of having `pods` and `flows` in separate folders? Because then what one sees inside a flow description is this `pods/encode.yml` and it seems missleading

Reply:
Would you elaborate more on your concerns about `pods/encode.yml`?

Reply:
It is very personal. But in the Flow `flow.yml` I see written.
```
uses: pods/exec.yml
...
```
I expect 'pods' to be a subfolder of where my `flow.yml` is. 

But it is very personal and quite irrelevant to the functionality

Reply:
Well, this is intended to make it clear they are configs for pods. I personally like it. Let's keep t it like this.

Reply:
I can see your point @JoanFM - when I moved the flows YAMLs the first time I also edited them to say `../pods/` bc I didn't know if it searched relative to it's own dir or the dir where `app.py` is run from. And then I realized it's the latter option and changed it back.

I guess my concern would be overcrowding the root dir of the example. While this isn't a problem with our "toy" examples (bc not many files), when building larger projects there should be a best practice for keeping things clean (also, folks are definitely going to be copying `flow-index.yml` to `flow-index.yml.old` and so on as they tinker, so it wouldn't just be 2 flow files crowding the dir)